package.json

```diff
@@ -23,21 +23,39 @@
   },
   "devDependencies": {
     "@google-cloud/datastore": "^1.0.2",
+    "@types/mocha": "^2.2.42",
+    "@types/node": "^8.0.26",
     "changelog-maker": "^2.2.2",
     "coveralls": "^2.11.8",
+    "del": "^3.0.0",
     "express": "^4.15.2",
     "glob": "^7.0.3",
     "google-auto-auth": "^0.7.0",
     "got": "^5.7.1",
+    "gts": "^0.3.0",
+    "gulp": "^3.9.1",
+    "gulp-coveralls": "^0.1.4",
+    "gulp-help": "^1.6.1",
+    "gulp-mocha": "^4.3.1",
+    "gulp-sourcemaps": "^2.6.1",
+    "gulp-tslint": "^8.1.2",
+    "gulp-typescript": "^3.2.2",
+    "gulp-util": "^3.0.8",
     "istanbul": "^0.4.2",
     "jshint": "^2.9.1",
+    "merge2": "^1.1.0",
     "mocha": "^3.0.0",
     "nock": "^9.0.0",
     "proxyquire": "^1.4.0",
+    "remap-istanbul": "^0.9.5",
     "request": "^2.81.0",
+    "source-map-support": "^0.4.16",
     "standard-version": "^4.2.0",
+    "through2": "^2.0.3",
     "timekeeper": "^1.0.0",
-    "tmp": "0.0.31"
+    "tmp": "0.0.31",
+    "tslint": "^5.7.0",
```

Comment:
If you use `gts check`, would this dependency still be needed?

Reply:
No, removed